Model/Validate.php

```diff
@@ -19,6 +19,7 @@
 use Magento\Framework\HTTP\Client\Curl;
 use Magento\Framework\Phrase;
 use Magento\Framework\Exception\LocalizedException;
+use Ekomi\EkomiIntegration\Helper\Data as DataHelper;
```

Comment:
Try not to keep things general. Here DataHelper is a general name for alias. You can name it like EkomiIntegrationHelper, etc or something like that which somewhat indicates the helper class purpose

Reply:
datahelper is default helper of module, magento's convention.

Reply:
Please ignore this point then. thanks for the clarity :)